community/kernel/src/main/java/org/neo4j/storageengine/api/StorageStatement.java

```diff
@@ -68,19 +69,21 @@
      * to place the cursor over the first item and then more calls to move the cursor through the selection.
      *
      * @param nodeId id of node to get cursor for.
+     * @param assertOpen to check if source transaction is still open on each property value fetch
```

Comment:
May I say that some comments are simply too detailed/bike-shed:y. Would you agree?